src/docs/tutorials/proximity-steering.md

```diff
@@ -0,0 +1,203 @@
+
+- Create a Health monitor.
+- Create pools for each region.
+- Set up the Proximity Load Balancer.
+
+**Prerequisites**
+
+- Have two or more identical services deployed in two or more different regions.
+
+
+    Duplicating a service can be done by right clicking and selecting **Duplicate**, opening its service settings and changing the region, then clicking **Deploy**.
+
+    The services you have deployed into the multiple regions should **only** have a Railway generated domain on them, it also helps to have the domains indicate the region.
+
+    Its recommended to use [shared variables](/guides/variables#shared-variables) or [reference variables](/guides/variables#referencing-another-services-variable) for duplicated services to keep variables in sync.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/v1722015743/docs/tutorials/proximity-load-balancing/region_services_u10ukp.png"
+alt="screenshot of two railway services in different regions"
+layout="responsive"
+width={890} height={435} quality={100} />
+
+- Have your desired domain setup with Cloudflare's nameservers, they have a general guide for that [here](https://developers.cloudflare.com/dns/zone-setups/full-setup/setup/).
+
+- Have **SSL/TLS** mode set to **Full**.
+
+    **SSL/TLS → Overview → Full**
+
+- Have **Always Use HTTPS** enabled.
+    
+    **SSL/TLS → Edge Certificates → Always Use HTTPS**
+    
+    This ensures that Railway avoids managing the insecure redirect, which would otherwise lead to an incorrect redirection to an upstream endpoint.
+
+- Have a `/health` or similar endpoint that returns a status code of 200.
+
+    This allows Cloudflare to check the health of our Railway services so they can handle region failover. As a bonus this can also be used on Railway to achieve [zero-downtime deployments](/reference/healthchecks).
+
+
+## 1. Creating a Health Monitor
+
+- Open the Load Balancing page.
+
+    **Traffic → Load Balancing**
+
+<Image src="https://res.cloudinary.com/railway/image/upload/v1722015860/docs/tutorials/proximity-load-balancing/load_balancing_page_yn5bm8.png"
+alt="screenshot of the load balancing page"
+layout="responsive"
+width={1060} height={555} quality={100} />
+
+- Click **Manage Monitors** and then **Create**.
+
+- Enter your desired name for this health monitor.
+
+- Choose **HTTPS** as the type.
+
+- Enter your health endpoint path
+
+    Example - `/health`
+
+- Leave Port 443 as the default.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/v1722015787/docs/tutorials/proximity-load-balancing/health_monitor_oty6pd.png"
+alt="screenshot of the cloudflare health monitor"
+layout="responsive"
+width={1060} height={315} quality={100} />
+
+- Click **Save**.
+
+## 2. Creating the Pools
+
+- Go back to the Load Balancing page.
+
+- Click **Manage Pools** and then **Create**.
+
+- Fill out the name and description and leave **Endpoint Steering** as its default, it will not be used with only a single endpoint.
```

Comment:
lil confusing.  in the screenshot, i see "Random" for this field.  Is that the default?  if so, maybe smth like ".. leave Endpoint Steering as the default of 'Random' .. "

Reply:
Yeah, random is the default, I will clarify